docs/DatabaseCatalogAPI.yaml

```diff
@@ -454,7 +454,36 @@ components:
             note:
               description: A note to clarify complex use cases for consumers.
               type: string
-
+            related_links:
+              description: >
+                A list of related links for the feed.
+              type: array
+              items:
+                $ref: '#/components/schemas/FeedRelatedLink'
+    FeedRelatedLink:
+      type: object
+      properties:
+        code:
+          description: >
+            A short code to identify the type of link.
+          type: string
+          example: next_1
+        description:
+          description: >
+            A description of the link.
+          type: string
+          example: The next feed URL after the current one
```

Comment:
Replace example text with: "The URL for the future version of the feed. This includes an upcoming service period."